tasks/build/download_node_builds.js

```diff
@@ -1,147 +1,134 @@
-module.exports = function (grunt) {
-  let { map, fromNode } = require('bluebird');
-  let { resolve, basename } = require('path');
-  let { createReadStream, createWriteStream } = require('fs');
-  let { createGunzip } = require('zlib');
-  let { Extract } = require('tar');
-  let { rename } = require('fs');
-  let checkHash  = require('check-hash');
-  let wreck = require('wreck');
+import { Promise } from 'bluebird';
```

Comment:
Bluebird provides promisify as a top level export, so you should be able to destructure directly into promisify here:

`import { promisify } from 'bluebird';